e2e-cypress/files/support/e2e.ts

```diff
@@ -0,0 +1,69 @@
+//For more details, you can follow this link:
+//https://docs.cypress.io/guides/end-to-end-testing/azure-active-directory-authentication#Microsoft-AAD-Application-Setup
+function loginViaAAD(username: string, password: string) {
+
+  //Go to your application URL and trigger the login.
+  cy.visit(Cypress.env('your_app_url'))
```

Comment:
Maybe I'm getting this wrong, but maybe calling just `cy.visit()` should do the trick here. Alternatively I would recommend to receive the actual URL via one an optional input of `loginViaAAD()` instead of using an arbitrary env var.